tests/dogtag/pytest-ansible/pytest/performance_test/test_cert_enrollment.py

```
@@ -74,7 +79,8 @@ def __init__(self, sn_uid):
         for enrollment_result in enrollment_results:
```

We should verify that `enrollment_results` is not empty.

---

Done

---

When would `enrollment_results` be empty? The server would have to return an empty array/dict. If there were an exception or something else, it would've failed.

---

I'm not quite sure, maybe it'll never happen, but it's a list so theoretically it could be empty (e.g. in case the client ignores the exception).

---

Added enrollment_results.__len__() == 0 if result is empty it will raise the exception.

---

Done